### PR TITLE
a few more docs about samples

### PR DESCRIPTION
**Pull request by petrelharp**
Following on a recent github thread, this might help clear up a little bit of confusion (although bigger things would help.

Could you have a look, @molpopgen?

### CODE REVIEW

**Review by jeromekelleher — APPROVED** [review]
LGTM

Let's wait on @molpopgen for review before merging

**Review by benjeffery — APPROVED** [review]

**Comment by petrelharp** [comment]
We've had two approvals (thanks!) and I've got too many things open; I'm merging this and Kevin can suggest changes elsewhere.

**Pull request merged**
